index.css

```diff
@@ -0,0 +1,739 @@
+}
+
+.headline > div > p {
+  font-size: 20px;
+  color: #344563;
+  line-height: 28px;
+  font-weight: 400px;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+
+.space-y-3 > section:nth-child(2n + 2) .card > div:nth-child(2) {
+  grid-column-start: 1;
+  grid-row-start: 1;
+  padding: 24px 0;
+}
+
+.card {
+  background-color: #fff;
+  padding: 24px;
+  border-radius: 24px;
+  margin: 0 8%;
+}
+
+.p-highlight {
+  list-style: none;
+  padding-inline-start: 0;
+  margin-top: 0;
+}
+
+.p-highlight li::before {
+  content: '\2022';
+  color: #bcc0c9;
+  font-weight: bold;
+  font-size: 1.5rem;
+  display: inline-flex;
+  width: 1rem;
+  margin-left: -1rem;
+  margin-right: 0.3rem;
+  vertical-align: sub;
+}
+
+.p-highlight li:first-child::before {
+  color: transparent;
+  margin-right: 0;
+  text-transform: uppercase;
+}
+
+.p-highlight li {
+  font-size: 18px;
+  color: #7a869a;
+}
+
+.skill-set {
+  padding-left: 24px;
+}
+
+.skill-set > ul > li > img {
+  width: 48px;
+  height: 48px;
+}
+
+.skill-set > ul > li > span {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+.skill-set > ul {
+  display: flex;
+  margin: 0 0 28px;
+  padding: 12px;
+}
+
+.skill-set > ul > li {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 120px;
+  width: 122px;
+  gap: 16px;
+  margin: 6px 12px;
+  padding: 12px;
+  border-radius: 8px;
+  color: #253858;
+  background-color: #f7f7f9;
+}
+
+.p-highlight li:first-child {
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #344563;
+}
+
+.list-tags > * {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  color: #6070ff;
+  font-size: 12px;
+  border-radius: 10px;
+}
+
+.connect {
+  color: #7f8cff;
+  margin-top: 10px;
+}
+
+.connect .social li {
+  list-style: none;
+  cursor: pointer;
+}
+
+.social > li > a > img {
+  max-width: 20px;
+  height: 19px;
+}
+
+.connect .social {
+  gap: 10px;
+}
+
+.connect #title {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.image-auto-h {
+  width: 100%;
+  height: auto;
+}
+
+.about-me {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  background-color: #fff;
+  padding: 108px 24px;
+  border-radius: 0 50px 0 0;
+  margin: 0 8%;
+}
+
+.btn-action {
+  padding: 12px 16px;
+  border-radius: 8px;
+  font-size: 17px;
+  color: #396df2;
+  font-weight: 500;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  background-color: #fff;
+}
+
+.btn-action:hover {
+  color: #fff;
+  background: #6070ff;
+}
+
+.btn-action:active {
+  background-color: #2230d2;
+}
+
+.tech {
+  height: 28px;
+  padding: 20px 0;
+  margin-left: 12px;
+  border-bottom: solid 1px #acb7c3;
+}
+
+.box {
+  flex-direction: row;
+}
+
+.lang {
+  border-bottom: none;
+  margin-top: 35px;
+  margin-bottom: 12px;
+}
+
+.tech h4 {
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.round-border-8 {
+  border-radius: 10px;
+}
+
+hr {
+  color: #dfe1e6;
+}
```

Comment:
- [ ] Kindly use the right color for your headings in the projects and the about me section. Also, you can add more gaps into your social icons:
figma:
![image](https://user-images.githubusercontent.com/91486791/222826125-2e3f9e84-c656-4a8a-91f6-3258e1b0deb6.png)
your design:
![image](https://user-images.githubusercontent.com/91486791/222826237-c1b0df13-9b2c-46fc-ab49-83bc286b11b8.png)